web/migrations/0019_update_charset_to_utf8mb4.py

```
@@ -27,9 +27,10 @@ def update_charset_mysql(apps, schema_editor):
             cursor.execute(
                 f"""
                 SELECT COLUMN_NAME, DATA_TYPE, CHARACTER_MAXIMUM_LENGTH
-                FROM INFORMATION_SCHEMA.COLUMNS
-                WHERE TABLE_NAME = '{table}'
-                AND (DATA_TYPE = 'varchar' OR DATA_TYPE = 'text' OR DATA_TYPE = 'longtext')
+            FROM INFORMATION_SCHEMA.COLUMNS
+            WHERE TABLE_SCHEMA = DATABASE()
+            AND TABLE_NAME = '{table}'
+            AND (DATA_TYPE = 'varchar' OR DATA_TYPE = 'text' OR DATA_TYPE = 'longtext')
             """
```

Ruff (0.8.2)

Possible SQL injection vector through string-based query construction

(S608)